dev/com.ibm.ws.microprofile.faulttolerance.2.0/src/com/ibm/ws/microprofile/faulttolerance20/state/impl/RetryStateImpl.java

```diff
@@ -171,6 +171,31 @@ public long getDelay() {
         public TimeUnit getDelayUnit() {
             return delayUnit;
         }
+
+        @Override
+        public String toString() {
+            return reason.toString();
```

Comment:
`reason` is not guaranteed to be non-null, you can't use it like this in a toString